feature/qos/ate_tests/bursty_traffic_test/metadata.textproto

```diff
@@ -27,7 +27,9 @@ platform_exceptions: {
     vendor: NOKIA
   }
   deviations: {
-    explicit_interface_ref_definition: true
```

Comment:
same here

ate testcase was deprecated and removed as per https://github.com/openconfig/featureprofiles/pull/2114 .

Hence, we no longer need this change for ATE

Reply:
Fixed branch conflict with removal of ATE changes.